contracts/modules/staking/StakingPool.sol

```diff
@@ -823,6 +830,21 @@ contract StakingPool is IStakingPool, SolmateERC721 {
     uint16 lastBucketId = coverAmounts[0].lastBucketId();
     uint currentBucket = block.timestamp / BUCKET_DURATION;
 
+    console.log("lastBucketId", lastBucketId);
+    console.log("currentBucket", currentBucket);
+    console.log("trancheCount", trancheCount);
+
+    /*
+      un-comment the line below to have the tx terminate
```

Comment:
there's a large disparity between lastBucketId and currentBucket which causes hundreds of iterations.

Seems like we'd need a lastBucketId saved somewhere